docs/tutorials/docker-compose.yml

```diff
@@ -2,160 +2,49 @@
 version: '2'
 services:
   zookeeper:
-    image: "confluentinc/cp-zookeeper:4.1.0"
-    hostname: zookeeper
```

Comment:
is this redundant because of the service name?

Reply:
I've been told that `hostname` isn't required. The `image` is still there just with a new version.